_posts/python/2016-07-26-get-slope-aspect-from-digital-elevation-model.md

```diff
@@ -22,162 +22,83 @@ With this data, we are able to use programs in order to analyze certain physical
 - Read in DEM data
 - Compute the slope for the entirety of the data
 - Compute the aspect for the entirety of the data
-- Output a PNG image visualizing the slope of the terrain
-- Output a PNG image visualizing the aspect of the terrain
+- Plot the slope of the terrain
+- Plot the aspect of the terrain
 
 ## Dependencies
 
-- NumPy
+- numpy
 - matplotlib
 - elevation
-- pyDEM
-
-Note that as of 2016-12-14, this tutorial uses a custom installation of pyDEM, which can be installed directly from GitHub from the command line via 
-
-```
-pip install git+git://github.com/mbjoseph/pydem.git
-```
+- richdem
 
 
 ```python
 import numpy as np
 import matplotlib
 import matplotlib.pyplot as plt
 import elevation
-from pydem.dem_processing import DEMProcessor
-from matplotlib.colors import ListedColormap
-from matplotlib import colors
-
-%matplotlib inline
-matplotlib.rcParams['figure.figsize'] = (10.0, 8.0)
+import richdem as rd
 ```
 
-## Download a digital elevation model
-
-Our first objective is acquire a DEM. We'll do this with the `elevation` module, which allows you to fetch data from the NASA SRTM mission via a bounding box. Here we draw a bounding box around Mt. Shasta in northern California, and use the command line interface to output a GeoTIFF. 
-
 
 ```python
-!eio clip -o Shasta-30m-DEM.tif --bounds -122.4 41.2 -122.1 41.5 
+matplotlib.rcParams['figure.figsize'] = (8, 5.5)
 ```
 
-    Your system is ready.
-    make: Nothing to be done for `download'.
-    make: Nothing to be done for `all'.
-    gdal_translate -q -co TILED=YES -co COMPRESS=DEFLATE -co ZLEVEL=9 -co PREDICTOR=2 -projwin -122.4 41.5 -122.1 41.2 SRTM1.vrt /Users/majo3748/Documents/earthlab/tutorials/python/Shasta-30m-DEM.tif
-
+## Download a digital elevation model
```

Comment:
we've been using all caps for our lesson titles BUT i wonder if taoti will advise us differently... if all of the tutorials are this way let's hold on fixing it until we talk with them!

Reply:
Got it - I'm curious to hear what they say too. I tend towards not using title case, as it can come off as a bit formal, but I am happy to change this to be consistent. Sounds like we can wait to figure out what they want, and merge this as is?

Reply:
let me ping @lherwehe  -- Lauren -- can you kind email Vince about title case vs just the first word capitalized? I know we spent a LOT of time changing the case throughout last year with our style guide. Can you kindly see what vince says about title case?